.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,7 @@
+Fixes issue #[Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issues #IssueNumber]
```

Comment:
Good suggestion. However we don't follow specific rules for branching. Contributors are free to use their own format. However commit message should start with `#issue_id <description>`